app/controllers/password_reset_controller.rb

```diff
@@ -0,0 +1,28 @@
+class PasswordResetController < ApplicationController
+
+  def create
+    user = User.find_by(email: params[:email])
+    user.send_password_reset if user
+    redirect_to login_url, :notice => "Email sent with password reset instructions."
+  end
+
+  def edit
+    @user = User.find_by(password_reset_token: params[:id])
```

Comment:
Though User.find_by(password_reset_token: params[:id]) will work but I want you to do this: User.find_by(password_reset_token: params[:token])